tests/src/test/java/alluxio/client/rest/JobMasterClientRestApiTest.java

```diff
@@ -136,6 +153,6 @@ private void waitForStatus(final long jobId, final Status status)
         Throwables.propagate(e);
       }
       return null;
-    }, WaitForOptions.defaults().setTimeoutMs(10 * Constants.SECOND_MS));
+    }, WaitForOptions.defaults().setInterval(10).setTimeoutMs(10 * Constants.SECOND_MS));
```

Comment:
why is this needed?

Reply:
I thought it made the test faster, but not really. I changed it back.